shared/chat/conversation/messages/reaction-tooltip/index.tsx

```diff
@@ -58,12 +58,11 @@ const ReactionTooltip = (props: Props) => {
           </Kb.Box2>
         )}
         <Kb.SectionList
-          alwaysBounceVertical={false}
```

Comment:
?

Reply:
Not sure why I removed this. Putting it back.